robots/breadboxBot.py

```diff
@@ -52,9 +55,11 @@ def __init__(self, period: float = 0.02) -> None:
         wpilib.SmartDashboard.setPersistent("Auto Distance 1")
         wpilib.SmartDashboard.setPersistent("Auto Distance 2")
 
+
     def getAutonomousCommand(self):
+        self.Position.getPosition()
```

Comment:
I'm pretty sure that self.Position.getPosition() should actually be a paramater in Autonomous()